spec.md

```diff
@@ -1339,6 +1338,9 @@ message Error {
       // information to detach the volume from one other node before
       // retrying with exponential backoff.
       MAX_ATTACHED_NODES = 8;
+
+      INVALID_NODE_ID = 9;
```

Comment:
Also suggested the recovery semantics for other error codes. The recovery semantics of the following errors are still missing since I'm not sure what behaviors are appropriate:

- ValidateVolumeCapabilitiesError
-- UNSUPPORTED_MOUNT_FLAGS
-- UNSUPPORTED_VOLUME_TYPE
-- UNSUPPORTED_FS_TYPE
- NodePublishVolumeError
-- UNSUPPORTED_MOUNT_FLAGS
-- UNSUPPORTED_VOLUME_TYPE
-- UNSUPPORTED_FS_TYPE
-- MOUNT_ERROR
- NodeUnpublishVolume
-- UNMOUNT_ERROR
- ProbeNodeError
-- BAD_PLUGIN_CONFIG
-- MISSING_REQUIRED_HOST_DEPENDENCY
- GetNodeIDError
-- BAD_PLUGIN_CONFIG
-- MISSING_REQUIRED_HOST_DEPENDENCY